messaging/Controllers/BundlesController.cs

```diff
@@ -46,7 +48,7 @@ public BundlesController(ILogger<BundlesController> logger, ApplicationDbContext
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-        public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdictionId, int _count, DateTime _since = default(DateTime), int page = 1)
+        public async Task<ActionResult<Bundle>> GetBundle(string jurisdictionId, int _count, string certificateNumber, string deathYear, DateTime _since = default(DateTime), int page = 1)

```

Comment:
To clear up the difference between these two functions, I think this one should remain "GetOutgoingMessageItems"